presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformQuantifiedComparisonApplyToScalarApply.java

```diff
@@ -164,7 +164,7 @@ private PlanNode rewriteQuantifiedApplyNode(ApplyNode node, QuantifiedComparison
                     node.getId(),
                     context.rewrite(node.getInput()),
                     subqueryPlan,
-                    Assignments.identity(minValue, maxValue),
+                    Assignments.identity(minValue, maxValue, countAllValue, countNonNullValue),
```

Comment:
They're in the following `Migrate TransformUncorrelatedScalarToJoin to Rule` commit, yet this change IMO deserves a separate commit. OTOH having a separate test in this commit asserting that some plan node has these specific assignments would be testing implementation details. The functionality that this code was introduced for in #6960 is thoroughly tested on query results level.